P20-P25.s

```diff
@@ -420,7 +420,7 @@ DE-GR-50	TC	2PHSCHNG
 #
 # ABORT MODES:  P23 ABORT IF MARKING SYSTEM OR EXTENDED VERB ACTIVE
 #
-# INPUT:  NONE REQURIES, NORMALLY CALLED BY P23
+# INPUT:  NONE REQUIRES, NORMALLY CALLED BY P23
```

Comment:
Based on the [source scan of page 570](http://www.ibiblio.org/apollo/ScansForConversion/Comanche055/0570.jpg), this line should read:

```
# INPUT:  NONE REQUIRED. NORMALLY CALLED BY P23
```